### ISSUE TITLE
Broken conda 1.29.1 package metadata

### ISSUE THREAD

**Issue by kukushking** [tlocke/pg8000#118]
Looks like something's gone wrong with Conda 1.29.1 package [noarch/pg8000-1.29.1-pyhd8ed1ab_0.tar.bz2](https://anaconda.org/conda-forge/pg8000/1.29.1/download/noarch/pg8000-1.29.1-pyhd8ed1ab_0.tar.bz2) metadata:

```
pg8000-0.0.0.dist-info
```
This is breaking my conda builds as my package depends on pg8000.

**Comment by BastianZim** [comment]
Unrelated, the error is conda-forge internal: https://github.com/conda-forge/pg8000-feedstock/pull/50

**Comment by tlocke** [comment]
This looks like it's fixed now, and it's also a duplicate of https://github.com/tlocke/pg8000/issues/113
